src/pngwriter.cc

```diff
@@ -765,6 +765,14 @@ int pngwriter::read(int x, int y, int colour) const
 
 	if(bit_depth_ == 16)
 	  {
+	     /* In these cases *256 is correct, because what we actually are
```

Comment:
yep .. I wanted to keep the style of the sourring indentation, because I just don't get the original indentation. I'm not even sure that tabs are 3 spaces... There are so many contradicting indentations. Tabs should be completely changed in another commit in my opinion.